kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ListAllReplicasActionIntegrationTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.integration.v3;
+
+import io.confluent.kafkarest.entities.v3.ListReplicasResponse;
+import io.confluent.kafkarest.entities.v3.ReplicaData;
+import io.confluent.kafkarest.entities.v3.Resource;
+import io.confluent.kafkarest.entities.v3.ResourceCollection;
+import io.confluent.kafkarest.integration.ClusterTestHarness;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ListAllReplicasActionIntegrationTest extends ClusterTestHarness {
+
+    private static final String TOPIC_1 = "topic-1";
```

Comment:
The formatting on this file looks off. We use 2-space ident, 4-space continuation.